examples/bench_cudamat.py

```diff
@@ -1,11 +1,12 @@
+from __future__ import print_function, division
```

Comment:
I believe print_function is needed for the tabular printing which I just fixed.
Is there a platform we'd like to support that uses python <2.6?